README.md

```diff
@@ -184,10 +184,6 @@ python mava/systems/ff_ippo.py env=rware env/scenario=tiny-4ag
 
 Additionally, we also have a [Quickstart notebook][quickstart] that can be used to quickly create and train your first Multi-agent system.
 
-## Advanced Usage 👽
```

Comment:
Ya @RuanJohn and I discussed it. I think if we want to bring it back at some point it would be nice, but right now it's not maintained and I'm not even sure if it works right now

Reply:
Indeed I agree on that front it has some issues within the evaluator as well, hpefully we can restore it in future 🤞